components/script/devtools.rs

```diff
@@ -125,7 +125,44 @@ pub fn handle_get_children(
     match find_node_by_unique_id(documents, pipeline, &node_id) {
         None => reply.send(None).unwrap(),
         Some(parent) => {
-            let children = parent.children().map(|child| child.summarize()).collect();
+            let is_whitespace = |node: &NodeInfo| {
+                node.node_type == 3 &&
+                    (node.node_value.is_none() ||
+                        node.node_value.clone().unwrap().trim().is_empty())
```

Comment:
```suggestion
                    (node.node_value.is_none() ||
                        node.node_value.clone().unwrap().trim().is_empty())
```

Could this use `map_or_else` to avoid an `unwrap` here?